File: lua/refactoring/tests/refactor/106/csharp/prompt_func_param_return/extract.expected.cs
namespace Test
{
    class Extract
    {
        public static int foo(int a) {
            Console.WriteLine("this is a test");
            int test = 1;
            Console.WriteLine(a);
            return test;
        }

        public static int simpleFunction(int a) {
            var test = foo(a);

            Console.WriteLine(test);

            return test;
        }
    }
}

File: lua/refactoring/tests/refactor/106/go/prompt_func_param_return/extract.expected.go
package main

import "fmt"

func foo(a int, test int) int {
    test_other := 1
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(idx, a)
    }
    return test_other
}

func simple_function(a int) (int, int) {
    var test int = 1
    test_other := foo(a, test)


    return test, test_other
}
